app/javascript/listings/components/ListingTagsField.jsx

```diff
@@ -0,0 +1,121 @@
+import { h } from 'preact';
+import { useEffect, useMemo, useState } from 'preact/hooks';
+import PropTypes from 'prop-types';
+import { useTagsField } from '../../hooks/useTagsField';
+import { TagAutocompleteOption } from '../../article-form/components/TagAutocompleteOption';
+import { TagAutocompleteSelection } from '../../article-form/components/TagAutocompleteSelection';
```

Comment:
I think we should move these to a more generic place, since they're now being used not only in the article form - can we move these components to within the `MultiSelectAutocomplete` folder?